### DISCUSSION TITLE
Student Dev Pack Application keeps getting rejected!

### DISCUSSION THREAD

**Discussion by rA9k** [community/community#56692] [GitHub Education]
### Hi everyone,

So i applied for it, but it was rejected at first because my name was different on github, so i changed it to my real name as it is on my provided document, but i still got rejected, then after searching this issue i tried few different things and kept getting rejected, then i changed my actual username (not display name) to my real name and now it says i've already submitted 10 requests. Can i please have a actual human review! each time when i got rejected i got email for it, and in it, it was my old display name in all emails. What can i do now?

**Comment by rA9k** [comment]
i submitted once more and again it got rejected, it just says this -
 What happened?
Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.

in the rejection email, my display name finally changed, but it still got rejected so what now?

**Comment by anderscain63** [comment]
@rohan-dinesh-chauhan I can provide you with some general suggestions on how to proceed in this situation:

- Contact GitHub Support: Reach out to GitHub's support team directly and explain your situation. They should be able to assist you with any account-related issues and guide you through the application process. You can usually contact them through the GitHub website or via email.

- Provide supporting documents: If you have any official documents that verify your student status, make sure to include them with your application or provide them to the support team when you contact them. This can help establish your eligibility for the Student Developer Pack.

- Double-check application details: Review your application thoroughly to ensure that all the information you've provided is accurate, including your name, educational institution, and any other required details. Pay close attention to consistency between your GitHub account and the information provided in your application.

**Comment by samus-aran** [comment]
Hi @rA9k 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

It would be preferable that a student is on campus when applying and thus able to upload a document but with remote learning that isn't always possible, so we request students submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. 

Let us know how you get on 👍
